polaris-react/src/components/Button/Button.scss

```diff
@@ -155,7 +155,7 @@
   background: var(--pc-button-color);
   border-width: 0;
   border-color: transparent;
-  box-shadow: var(--p-shadow-button), var(--p-shadows-inset-button);
```

Comment:
`var(--p-shadows-inset-button)` gets migrated to `var(--p-shadow-none)` so it is not needed here anymore.